samples/local_attestation/localattestation.edl

```diff
@@ -2,6 +2,11 @@
 // Licensed under the MIT License.
 
 enclave {
+    from "openenclave/edl/attestation.edl" import *;
+    from "openenclave/edl/logging.edl" import *;
+    from "openenclave/edl/syscall.edl" import *;
+    from "platform.edl" import *;
```

Comment:
It reads as if `platform.edl` is a user-written EDL, since for other system edl's we require `openenclave/edl/` qualification.

Reply:
This is to allow someone to use the same edl file for both SGX and OP-TEE by just changing build files.

We could get rid of this by implementing https://github.com/openenclave/openenclave/issues/2833

Now that edger8r is C++, implementing this might be a bit easier.